stdlib/gc.mli

```diff
@@ -385,3 +386,79 @@ val create_alarm : (unit -> unit) -> alarm
 val delete_alarm : alarm -> unit
 (** [delete_alarm a] will stop the calls to the function associated
    to [a].  Calling [delete_alarm a] again has no effect. *)
+
+(** [Memprof] is an sampling engine for allocated words. The blocks
+
```

Comment:
Why a blank line here? Is it on purpose?

Reply:
I have no idea why this new line is here. Please feel free to remove it.